cmake/rapids_config.cmake

```diff
@@ -11,6 +11,9 @@
 # or implied. See the License for the specific language governing permissions and limitations under
 # the License.
 # =============================================================================
+set(rapids-cmake-repo bdice/rapids-cmake)
+set(rapids-cmake-branch cccl-2.8.0)
+
```

Comment:
Revert this testing change before merging. This PR will need to merge immediately after https://github.com/rapidsai/rapids-cmake/pull/793.
```suggestion
```